templates/app/assets/stylesheets/spree/frontend/base/_utilities.scss

```diff
@@ -0,0 +1,3 @@
+.d-hide {
+  display: none;
+}
```

Comment:
I placed the `utilities` SCSS file under base directory. It was under `app/assets/stylesheets/general` in Solidus Demo.

Reply:
I think this makes more sense, thank you for the correction